contentful/client.py

```diff
@@ -405,8 +405,16 @@ def _contentful_user_agent(self):
             'name': 'python',
             'version': platform.python_version()
         }
+
+        os_name = platform.system()
+        if os_name == 'Darwin':
+            os_name = 'macOS'
+        elif not os_name or os_name == 'Java':
+            os_name = 'Unknown'
```

Comment:
lol, I just made exactly the same comment over here https://github.com/contentful/contentful-management.py/pull/8#discussion_r118660227